Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Last-Update: 2013-03-30
Forwarded: https://github.com/jpirko/libteam/pull/4
Description: fixes FTBFS with format hardening

--- a/utils/teamdctl.c
+++ b/utils/teamdctl.c
@@ -61,9 +61,9 @@
 
 #define pr_err(args...) fprintf(stderr, ##args)
 #define pr_outx(verb_level, args...) \
 	if (verb_level <= g_verbosity) { \
-		fprintf(stdout, g_indent_str); \
+		fprintf(stdout, "%s", g_indent_str); \
 		fprintf(stdout, ##args); \
 	}
 #define pr_out(args...) pr_outx(DEFAULT_VERB, ##args)
 #define pr_out2(args...) pr_outx(VERB2, ##args)
