Last-Update: 2013-10-12
Forwarded: not-needed
Origin: upstream, https://github.com/jpirko/libteam/commit/cdb505ba5e0688fa13b227e192c50caa72cf9562
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Description: do not install example binaries

--- a/examples/Makefile.am
+++ b/examples/Makefile.am
@@ -5,9 +5,9 @@
 AM_CFLAGS = -I${top_srcdir}/include
 
 LDADD = $(top_builddir)/libteam/libteam.la
 
-bin_PROGRAMS=team_manual_control team_monitor
+noinst_PROGRAMS=team_manual_control team_monitor
 team_manual_control_SOURCES=team_manual_control.c
 team_monitor_SOURCES=team_monitor.c
 
 EXTRA_DIST = python
