Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Last-Update: 2012-12-04
Forwarded: no
Description: spelling correction(s)

--- a/teamd/teamd.c
+++ b/teamd/teamd.c
@@ -1438,9 +1438,9 @@
 	}
 
 	daemon_retval_send(0);
 
-	teamd_log_info(PACKAGE_VERSION" sucessfully started.");
+	teamd_log_info(PACKAGE_VERSION" successfully started.");
 
 	err = teamd_run_loop_run(ctx);
 
 	teamd_log_info("Exiting...");
